src/components/Person/Person.jsx

```diff
@@ -1 +1,23 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  let partnerString = '';
+
+  if (person.isMarried) {
+    if (person.sex === 'm') {
+      partnerString = `${person.partnerName} is my wife`;
+    } else {
+      partnerString = `${person.partnerName} is my husband`;
+    }
+  } else {
+    partnerString = 'I am not married';
+  }
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+      {person.age !== undefined && (
```

Comment:
```suggestion
      {person.age && (
```
The better way doesn't use type condition.